jenkins/ci.suse.de/templates/cloud-update-ci-trigger-template.yaml

```diff
@@ -0,0 +1,19 @@
+- job-template:
+    name: 'cloud-update-ci-{instance}-trigger'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H */3 * * *'
```

Comment:
hmm, that doesn't respect existing PRs, so those changes would get nuked every 3 hours. not good. ideally I would only like to deploy it only on a git merge. do we have any of the potential plugins for that installed in jenkins ?